### ISSUE TITLE
Low Success Rate/Qscore for turning_on_radio Task After Finetuning with GR00T-N1.6-BEHAVIOR1k

### ISSUE THREAD

**Issue by steinate** [NVIDIA/Isaac-GR00T#515]
I am finetuning the turning_on_radio task using the official script examples/BEHAVIOR/finetune_BEHAVIOR.sh with the pretrained model GR00T-N1.6-BEHAVIOR1k (from https://huggingface.co/nvidia/GR00T-N1.6-BEHAVIOR1k). After training up to 150,000 steps, the evaluation results show a much lower success rate/Q score than the values (0.70) reported in the project’s README. And there is no video recorded so I have no idea about my policy details.
## Training Logs
<img width="2174" height="708" alt="Image" src="https://github.com/user-attachments/assets/bb9ac5c6-ccec-4923-9b4c-b1de805a5ace" />

## Evaluation Results
```
Collecting 10 episodes took 2297.4847469329834 seconds                                                                                                               
Video saved to:  None                                                                                                                                                
results:  ('sim_behavior_r1_pro/turning_on_radio', [False, False, False, False, False, False, False, False, True, False], {'task_progress': [66.66666666666666, 66.66
666666666666, 66.66666666666666, 33.33333333333333, 33.33333333333333, 66.66666666666666, 66.66666666666666, 66.66666666666666, 100.0, 33.33333333333333], 'q_score':
 [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0], 'valid': [True, True, True, True, True, True, True, True, True, True]})                                         
success rate:  0.1                                                                                                                          
```

## Training Script
```
set -x -e

export NUM_GPUS=8

torchrun --nproc_per_node=$NUM_GPUS --master_port=29500 \
    gr00t/experiment/launch_finetune.py \
    --base_model_path models/GR00T-N1.6-BEHAVIOR1k \
    --dataset_path data/behavior-dataset/sim_behavior_r1_pro.task-0000_turning_on_radio \
    --embodiment_tag BEHAVIOR_R1_PRO \
    --num_gpus $NUM_GPUS \
    --output_dir ./output/BEHAVIOR_finetune \
    --save_steps 1000 \
    --save_total_limit 5 \
    --max_steps 150000 \
    --warmup_ratio 0.05 \
    --weight_decay 1e-5 \
    --learning_rate 1e-4 \
    --use_wandb \
    --global_batch_size 1024 \
    --color_jitter_params brightness 0.3 contrast 0.4 saturation 0.5 hue 0.08 \
    --dataloader_num_workers 4 \
    --state_dropout_prob 0
```

**Comment by zjuluolun** [comment]
Mark. I have the same question.
BTW, how many action steps you use when performing inference. I find this parameter important to the final score.

**Comment by seunghub** [comment]
Hi, thanks for your work on this project.

I tested the official finetuned checkpoint, but I’m getting a success rate of 0 when turning on the radio, and the average task progress is only around 20.

I was wondering if you have benchmark results or observed performance for the official checkpoint on this task. It would be helpful to compare and understand whether this behavior is expected or if something might be wrong on my side.

Thanks in advance!

**Comment by xxxxyu** [comment]
Hi, similar issues here. My findings so far:

1. The official fine-tuning script seems specific for multi-task (50-task) fine-tuning.
2. When using it for a single task (e.g., turning_on_radio), training loss looks normal but eval performance degrades compared to the base BEHAVIOR-1k checkpoint (e.g., from 43% to below 30%). It is likely due to suboptimal hyperparameters (lr too high & too many steps).
3. The evaluation itself has some randomness (e.g., env initialization), which increases the difficulty to reproduce GR00T's official results.

If anyone has progress reproducing single-task results (https://github.com/NVIDIA/Isaac-GR00T/blob/main/examples/BEHAVIOR/README.md#1-individual-task-post-training-optional), please share wit us. Thanks!

It would also be great if the GR00T team could provide more guidance on single-task post-training hyperparameters. Thanks for open-sourcing this! @youliangtan (sorry if I pinged the wrong person, inferred from the N1.6 release commit)
